src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -587,6 +587,8 @@ studio.cors.credentials: true
 # Value for the Access-Control-Max-Age header
 studio.cors.maxage: -1
 
+# The active environment for multi environment configuration, e.g. qa, prod, dev
+studio.configuration.environment.active: ${env:CRAFTER_ENVIRONMENT}
```

Comment:
This should go in the override in craftercms instead of this repo

Reply:
Should this have something basic/foundational that's overridden by the one in `craftercms`?

Reply:
It's never been there